src/md-to-html.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -e
+
+throw() { 
+  echo "$@" 1>&2
+  exit 1
+}
+
+# Validate arguments
+if [ $# -eq 0 ]
+  then
+    throw "No arguments supplied. Please specify md files to convert as first argument and output directory as second argument"
+fi
+
+
+# ensure showdown exists in path
+if ! [ -x "$(command -v showdown)" ]; then
+  throw "Showdown is not installed. Please install showdown and ensure it is made available at ./node_modules/.bin/showdown"
+fi
+
+# save args to variables
+inputFiles=$1
+destDir=$2
+
+# create dir, if destination directory does not exist
+if [ ! -d "$destDir" ]; then
+  mkdir -p "$destDir"
+fi
+
+# convert files
+for file in $inputFiles; 
+  do
+    filename=$(basename -- "$file")
+    extension="${filename##*.}"
+    filename="${filename%.*}"
+
+    if [ ! "$extension" = "md" ]
```

Comment:
Might be nice to move this check out of the loop so the program doesn't convert half of the files and error on the last one.